internal/test/windows/windows.go

```diff
@@ -81,6 +87,45 @@ func (w *Windows) CopyFile(filePath, remoteDir string) error {
 	return nil
 }
 
+// CopyDirectory copies the files from the directory on the local host to the remote Windows VM
+// This does not copy nested directories
+func (w *Windows) CopyDirectory(localDir string, remoteDir string) error {
```

Comment:
nit : maybe a better name would be `TransferDirectory` for this func, to be in sync with the flag

Reply:
But this would put it in out of sync with `CopyFile`. I suggest leaving it as `CopyDirectory`.